pkg/koordlet/util/system/kidled_util.go

```diff
@@ -198,9 +197,31 @@ func SetKidledUseHierarchy(useHierarchy uint8) error {
 	write.Flush()
 	return nil
 }
+
+func GetKidledColdBoundary() int {
+	return kidledColdBoundary
+}
+
+func SetKidledColdBoundary(boudary int) {
+	kidledColdBoundary = boudary
+}
+
 func NewDefaultKidledConfig() *KidledConfig {
 	return &KidledConfig{
-		ScanPeriodInseconds: 5,
-		UseHierarchy:        1,
+		ScanPeriodInseconds: defaultKidledScanPeriodInseconds,
+		UseHierarchy:        defaultKidledUseHierarchy,
+		KidledColdBoundary:  defaultKidledColdBoundary,
+	}
+}
+
+func sum(nums ...[]uint64) uint64 {
+	var total uint64
+	for _, v := range nums {
+		for i, num := range v {
```

Comment:
recommend renaming the func as `sumUint64Slice`.

As for the inner loop, how about
```go
for i := kidledColdBoundary; i < len(v); i++ {
    // ...
}
```

Reply:
ok